ownCloudAppShared/User Interface/More/MoreViewHeader.swift

```diff
@@ -86,6 +86,12 @@ open class MoreViewHeader: UIView {
 
 	private func render() {
 		cssSelectors = [.more, .header]
+		
+		let secureView = SecureTextField().secureContainerView
```

Comment:
The suggestions about unifying the code to inject a secure container that I made in `BrowserNavigationViewController.swift` would also apply here.